framework/test/jest/specs/unit/controller/channels/child_process_channel.spec.js

```diff
@@ -0,0 +1,622 @@
+const axon = require('axon');
+const axonRpc = require('axon-rpc');
+
+jest.mock('eventemitter2');
+jest.mock('../../../../../../src/controller/event', () =>
+	jest.fn().mockImplementation(name => ({
+		name,
+		module: 'moduleAlias',
+		key: () => 'aKey',
+		serialize: () => 'serialized',
+	}))
+);
+
+axonRpc.Client = function() {
+	return {
+		call: jest.fn(),
+	};
+};
+axonRpc.Server = function() {
+	return {
+		expose: jest.fn(),
+	};
+};
+
+const EventEmitter2 = require('eventemitter2');
```

Comment:
import before mocking?

Reply:
Moved imports before mocking. But had to leave manual mocking of axon-RPC Client and Server as ChildProcessChannel needs to have those mocked when requiring them.

Reply:
ah got it! 👍